unpack.Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o unpack ./cmd/unpack/
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/static:debug
```

Comment:
Is this change intended to be permanent?

Reply:
If we're copying over the unpack executable to the bundle.Spec.Image container image, which is scratch-based (i.e. no shell), then we need the unpack executable to have a shell so we can fire off `cp` commands for copying over the unpack binary in an initContainer to the container image that unpacks the bundle.Spec.Image.

Reply:
Oh right. Thanks.

Reply:
Maybe add a comment about why we need `:debug` since it isn't obvious just looking at the Dockerfile.